model/ServerHandler.php

```diff
@@ -0,0 +1,167 @@
+<?php
+# $Id: DomainHandler.php 2016-03-11 00:16:00Z jan-frederik_rieckers $
+
+/**
+ *
+ */
+class ServerHandler extends PFAHandler{
+
+    protected $db_table = 'server';
+    protected $id_field = 'server';
+
+    protected function validate_new_id() {
+        return true;
```

Comment:
I'm not sure if we want or need to enforce a full host name, but I'd recommend to at least check for "not empty" and "not only whitespace" ;-)